### ISSUE TITLE
Sick Rage Manage Torrents Shows up as Phishing site on Safari & doesnt load 

### ISSUE THREAD

**Issue by XxUnkn0wnxX** [SiCKRAGETV/sickrage-issues#2964]
now i don't have this issue when i load uTorrent web guy directly or via sickrage in chrome/firefox

so not sure why this happens only in Safari:

http://puu.sh/koe1s.png
http://puu.sh/koe25.png

**Comment by OmgImAlexis** [comment]
Without knowing what url you're using for utorrent, what version you're on etc. we can't really help. Please read the guidelines about posting an issue. https://github.com/SiCKRAGETV/sickrage-issues

**Comment by XxUnkn0wnxX** [comment]
i know the URL: localhost:8082/gui

but in sickrage config i have just the localhost:8082 otherwise with localhost:8082/gui it fails to register.

**Comment by neoatomic** [comment]
With manage torrents, sickrage opens the uTorrent webui IINSIDE of the sickrage webui. That can be a reason for a Malware/security app to flag it. (As it basically redirects you.)

Dont think we can do anything about that. (other than opening it in a new window.?)

**Comment by OmgImAlexis** [comment]
@neoatomic what if we use the iframe but with sandbox enabled? I think that may be why Safari has a problem with it.

**Comment by neoatomic** [comment]
Not the expert here, but for sure we can give that a try and see if it works. :)

**Comment by OmgImAlexis** [comment]
@neoatomic I worked it out, it's because we access utorrent using `http://user:password@ip_address:port/gui.`

**Comment by OmgImAlexis** [comment]
There is literally nothing we can do about this. All major browsers are removing support for this, eventually it won't even load in an iframe anymore. We're going ot have to remove this feature. The only thing that may work would be to proxy the page through SR which is just more hassle than it's worth.

@miigotu

**Comment by miigotu** [comment]
@OmgImAlexis it has long been requested to remove that anyways. Many people complained so we kept it for now but it is a big security hole, and only works sometimes.

**Comment by OmgImAlexis** [comment]
Well as of one of the last Chrome updates they're getting ready to block it so it'll have to be removed sooner or later.

**Comment by WebSpider** [comment]
Is there no other way to do this? It seems to be a feature people like (and I can imagine they do, it's a complete interface in one webpage)

**Comment by miigotu** [comment]
Not really. As it sits right now if you remotely access it your user:pass would be sent over the network in plain text (even if you use https)

**Comment by OmgImAlexis** [comment]
@miigotu any chance we could proxy it through SR? Instead of an iframe to utorrent we could do an iframe to SR that loads the page via python.

**Comment by miigotu** [comment]
Im sure we could always DO it, but making it work with every client....

**Comment by OmgImAlexis** [comment]
Well it's either proxy it or remove it, Chrome and all webkit based browsers are removing it by the end of the year.

**Comment by miigotu** [comment]
I'll male an attempt at loading it right through requests next chance I get.
